python/util.py

```diff
@@ -22,6 +22,97 @@
 import array
 import struct
 from uuid import getnode as get_mac
+import winreg as wreg
```

Comment:
Might want to detail somewhere (if it isn't already) that this will be python3 only